WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsPluginNotActivatedView.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
+struct InPersonPaymentsPluginNotActivatedView: View {
+    let onRefresh: () -> Void
+
+    var body: some View {
+        VStack {
+            Spacer()
+
+            VStack(alignment: .center, spacing: 42) {
+                Text(Localization.title)
+                    .font(.headline)
+                    .multilineTextAlignment(.center)
+                Image(uiImage: .paymentsPlugin)
+                    .resizable()
+                    .scaledToFit()
+                    .frame(height: 108.0)
+                Text(Localization.message)
+                    .font(.callout)
+                    .multilineTextAlignment(.center)
```

Comment:
We can consolidate this at the end, but in #4750 I realized we can put the `multilineTextAlignment` once in the `VStack` and all the Text views inside it will inherit that.

Reply:
I've just pushed an update to apply the style to the Stack. It takes just one minute, and it's one less thing to worry about.

<img src="https://user-images.githubusercontent.com/2722505/128740450-63fdf48d-bc62-4c75-8b3b-6e2d0501a99a.png" width="350"/>

I'll merge as soon as CI is ✅